game/src/main/scala/io/aigar/game/Food.scala

```diff
@@ -0,0 +1,16 @@
+package io.aigar.game
+
+import io.aigar.game.serializable.Position
+
+class Food {
+  var regular = List(new Position(12, 12))
+  var silver = List(new Position(112, 112))
+  var gold = List(new Position(212, 212))
+
+  def state = {
+    serializable.Food(regular,
+                      silver,
+                      gold
```

Comment:
nitpick, but why is the `)` on the other line? :) Doesn't really matter, though.